src/extra/Markdown.tsx

```diff
@@ -13,10 +13,39 @@ const markedOptions = {
 	gfm: true,
 	breaks: true,
 	headerIds: false,
-	sanitize: true,
+	// Input text is sanitized using `sanitize-html` prior to being transformed by
+	// `marked`
+	sanitize: false,
 };
 
-const render = memoize((text: string = '') => marked(text, markedOptions));
+const sanitizerOptions = {
```

Comment:
Maybe we can add a test or two to check if the input is properly sanitized.

Reply:
Good idea